include/mbedtls/base64.h

```diff
@@ -42,7 +42,9 @@ extern "C" {
 /**
  * \brief          Encode a buffer into base64 format
  *
- * \param dst      destination buffer
+ * \param dst      destination buffer (can be NULL for checking size, also src
+ *                 and dst buffers may overlap: int this case inplace encoding
+ *                 takes place and src buffer is overwritten)
```

Comment:
Please use full sentences, correct punctuation, and applicable formatting. Older documentation sometimes takes liberties, but we fix those when we touch them.

> Destination buffer. This can be `\c NULL` to obtain the required buffer size.
> The \p src and \p dst buffers may overlap.

Reply:
Hopefully it should be ok now.